lib/media/track/sender.js

```diff
@@ -52,7 +52,13 @@ class MediaTrackSender extends MediaTrackTransceiver {
    */
   _reemitMediaStreamTrackEvents(mediaStreamTrack = this._track) {
     const { _eventsToReemitters: eventsToReemitters, _track: track } = this;
-    eventsToReemitters.forEach((reemitter, event) => mediaStreamTrack.addEventListener(event, reemitter));
+    if (mediaStreamTrack.addEventListener) {
+      eventsToReemitters.forEach((reemitter, event) => mediaStreamTrack.addEventListener(event, reemitter));
+    } else {
+      eventsToReemitters.forEach((reemitter, event) => {
+        mediaStreamTrack[`on${event}`] = reemitter;
+      });
+    }
     if (track !== mediaStreamTrack) {
       eventsToReemitters.forEach((reemitter, event) => track.removeEventListener(event, reemitter));
```

Comment:
Do you also need to wrap `removeEventListener` similar to what you're doing above for `addEventListener`?

Reply:
Correct 👍